website/content/en/docs/concepts/nodepools.md

```diff
@@ -72,7 +72,7 @@ spec:
           operator: In
```

Comment:
Can you update these docs to be in the preview directory as opposed to the `docs` directory. I admit it's a bit confusing (and we need to eventually change it) but updates to the docs go into the preview directory ahead of them being formally released in docs and in the latest version